endpoint/OBDComp/emulator/script.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SUDO=false
+if which sudo &>/dev/null; then
+	SUDO=true
+fi
+
```

Comment:
If `vcan0` interface is not available via kernel configurations, you should alert.